modules/custom-status/rest/custom-status-endpoint.php

```diff
@@ -260,17 +276,24 @@ public static function handle_update_status( WP_REST_Request $request ) {
 
 		// get status_name & status_description
 		$args = [
-			'name'               => $status_name,
-			'description'        => $status_description,
-			'slug'               => $status_slug,
-			'is_review_required' => $status_is_review_required,
+			'name'              => $status_name,
+			'description'       => $status_description,
+			'slug'              => $status_slug,
+			'required_user_ids' => $status_required_user_ids,
 		];
 
 		// ToDo: Ensure that we don't do an update when the name and description are the same as the current status
-		$update_status_result = $custom_status_module->update_custom_status( $term_id, $args );
+		$updated_status = $custom_status_module->update_custom_status( $term_id, $args );
+
+		/**
+		* Add additional data to status terms used in the UI
+		*
+		* @param WP_Term $updated_status An updated status term.
+		*/
+		$updated_status = apply_filters( 'vw_custom_status_ui_additions', $updated_status );
```

Comment:
I think it'd be better to move this into the `update_custom_status` so that you get the entire object after a save or update.

Reply:
Note: I do like the idea of adding the username instead of just the IDs as it saves a call. Adding it to another transient meta might make sense so it's known that it's not actually stored in the DB. Overriding the `get_term` and `get_terms` call should do the trick.